# Study 888 — CLO AAA Carry 🔒

**Does the senior slice of a CLO pay a *real* pickup over cash and same-rated corporates?**

A **AAA-rated CLO tranche** sits at the top of a collateralised-loan-obligation waterfall:
first to be paid, last to take losses, protected by a thick cushion of subordinate tranches.
It pays a spread over cash and over same-rated IG corporate bonds — the story goes — as
compensation for *structural complexity* (a securitisation few desks underwrite), not for
credit risk (realized senior-tranche defaults are ~nil). Since 2020-10 it trades in a liquid,
floating-rate ETF (**JAAA**; **ICLO** from 2022-12), so it carries almost no *duration* either.

We test that carry **excess-of-cash** (minus BIL) against IG corporates (**LQD**), Treasuries
(**IEF**), and — the sharp control — the *un-tranched* leveraged-loan collateral itself
(**BKLN**). 2020-10-20 → 2026-06-30.

*Numbers below are the frozen headline (`docs/results.md`); the live cells run the fast
synthetic control. **Short history:** JAAA ~5.7y, ICLO ~3.5y — ONE rate cycle, NO CLO
credit-stress event in-sample, so realized Sharpes are an upper bound (named on the Signal
axis). This is distinct from **[614-clo-equity-yield](../../614-clo-equity-yield/)**, the
risky first-loss BOTTOM of the same stack.*


## 1. The idea in one line

Buy the *safest* slice of a pool of leveraged loans — the AAA tranche, shielded by everything below it — and you still get paid a spread over T-bills for the sheer hassle of the structure. Because the tranche floats, there's almost no interest-rate risk; because it's senior, there's almost no default risk. The question is whether that 'complexity spread' is a *real* risk-adjusted edge or just repackaged cash.

In [1]:
R = {'asof': '2026-06-30', 'fp': '3568185876b2', 'n_rows': 1631, 'jaaa_start': '2020-10-20', 'jaaa_n': 1429, 'jaaa_exc': 1.38, 'jaaa_vol': 1.63, 'jaaa_sharpe': 0.84, 'jaaa_lo': 0.09, 'jaaa_hi': 1.72, 'jaaa_t': 2.33, 'jaaa_dd': -2.6, 'jaaa_fracneg': 0.015, 'jaaa_bpsday': 0.55, 'iclo_start': '2022-12-12', 'iclo_n': 889, 'iclo_exc': 2.14, 'iclo_vol': 2.43, 'iclo_sharpe': 0.88, 'iclo_lo': 0.22, 'iclo_hi': 2.31, 'iclo_t': 3.08, 'iclo_dd': -3.46, 'bkln_exc': 1.82, 'bkln_vol': 7.71, 'bkln_sharpe': 0.24, 'bkln_t': 0.53, 'bkln_dd': -24.17, 'lqd_exc': -1.09, 'lqd_vol': 10.27, 'lqd_sharpe': -0.11, 'lqd_t': -0.27, 'lqd_dd': -24.95, 'ief_exc': -2.42, 'ief_vol': 7.46, 'ief_sharpe': -0.33, 'ief_t': -0.9, 'ief_dd': -23.92, 'h2h_lqd': 4.1, 'h2h_lqd_t': 1.23, 'h2h_ief': 5.85, 'h2h_ief_t': 2.0, 'h2h_bkln': -0.86, 'h2h_bkln_t': -0.6, 'zirp_n': 427, 'zirp_exc': 0.0, 'zirp_sharpe': 0.0, 'zirp_t': 0.0, 'high_n': 875, 'high_exc': 1.84, 'high_sharpe': 1.37, 'high_t': 2.96, 'cost1_net': 1.35, 'cost1_sharpe': 0.83, 'cost1_t': 2.28, 'cost12_net': 1.02, 'cost12_sharpe': 0.62, 'cost12_t': 1.72, 'rel_gross': 4.1, 'rel_charge': 1.12, 'rel_net': 2.98, 'rel_sharpe': 0.35, 'rel_t': 0.89, 'null_sharpe_mean': 0.03, 'null_sharpe_sd': 0.34, 'null_fire': 0, 'planted_exc': 1.77, 'planted_sharpe': 1.73, 'planted_t': 3.9, 'planted_lo': 0.86, 'planted_hi': 2.58}
print(f"JAAA excess-of-cash : {R['jaaa_exc']:+.2f}%/yr on {R['jaaa_vol']:.2f}% vol "
      f"-> Sharpe {R['jaaa_sharpe']:+.2f}  (95% CI [{R['jaaa_lo']:+.2f}, {R['jaaa_hi']:+.2f}], HAC t {R['jaaa_t']:+.2f})")
print(f"  its worst drawdown in ~5.7y: {R['jaaa_dd']:+.2f}%  (a floating senior tranche barely wobbles)")
print(f"ICLO (2nd AAA-CLO fund): Sharpe {R['iclo_sharpe']:+.2f}  (HAC t {R['iclo_t']:+.2f}) -- an independent confirm")

JAAA excess-of-cash : +1.38%/yr on 1.63% vol -> Sharpe +0.84  (95% CI [+0.09, +1.72], HAC t +2.33)
  its worst drawdown in ~5.7y: -2.60%  (a floating senior tranche barely wobbles)
ICLO (2nd AAA-CLO fund): Sharpe +0.88  (HAC t +3.08) -- an independent confirm


## 2. The tell — the AAA tranche vs the *raw loans* it's built from

The cleanest control isn't corporate bonds — it's **BKLN**, the same leveraged loans held *un-tranched* and below investment grade. If seniority/tranching is doing real work, the AAA slice should deliver a far better *risk-adjusted* return than the whole.

In [2]:
print(f"JAAA  : Sharpe {R['jaaa_sharpe']:+.2f}   maxDD {R['jaaa_dd']:+6.2f}%   (senior AAA tranche)")
print(f"BKLN  : Sharpe {R['bkln_sharpe']:+.2f}   maxDD {R['bkln_dd']:+6.2f}%   (un-tranched leveraged loans)")
print(f"LQD   : Sharpe {R['lqd_sharpe']:+.2f}   maxDD {R['lqd_dd']:+6.2f}%   (IG corporates, ~8y duration)")
print(f"IEF   : Sharpe {R['ief_sharpe']:+.2f}   maxDD {R['ief_dd']:+6.2f}%   (7-10y Treasuries)")
print('--> the AAA slice earns MORE per unit of risk than the loans it is carved from,')
print('    and dodged the -24% drawdowns that duration (LQD/IEF) and raw loans (BKLN) took.')

JAAA  : Sharpe +0.84   maxDD  -2.60%   (senior AAA tranche)
BKLN  : Sharpe +0.24   maxDD -24.17%   (un-tranched leveraged loans)
LQD   : Sharpe -0.11   maxDD -24.95%   (IG corporates, ~8y duration)
IEF   : Sharpe -0.33   maxDD -23.92%   (7-10y Treasuries)
--> the AAA slice earns MORE per unit of risk than the loans it is carved from,
    and dodged the -24% drawdowns that duration (LQD/IEF) and raw loans (BKLN) took.


## 3. Is the detector honest? A live synthetic control

We plant a steady +1.2%/yr excess-of-cash carry in a seeded toy world (plus a high-vol 'duration' decoy that earns *nothing* extra) and check the detector recovers it — and stays silent on the null (`carry=0`). No network.

In [3]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from clo_aaa import data, strategy as st
planted = data.synthetic_world(carry_annual=0.012, seed=888).rename(columns={'cash':'BIL','carry':'CARRY','dur':'DUR'})
nullw   = data.synthetic_world(carry_annual=0.0,   seed=888).rename(columns={'cash':'BIL','carry':'CARRY','dur':'DUR'})
p = st.carry_stats(planted, 'CARRY', cash='BIL', n_boot=400)
z = st.carry_stats(nullw,   'CARRY', cash='BIL', n_boot=400)
print(f"planted carry: excess {p['excess_ann_pct']:+.2f}%/yr  Sharpe {p['sharpe']:+.2f}  HAC t {p['t_hac']:+.2f}  (should light up)")
print(f"null world   : excess {z['excess_ann_pct']:+.2f}%/yr  Sharpe {z['sharpe']:+.2f}  HAC t {z['t_hac']:+.2f}  (should be ~0)")

planted carry: excess +1.77%/yr  Sharpe +1.73  HAC t +3.90  (should light up)
null world   : excess +0.57%/yr  Sharpe +0.56  HAC t +1.26  (should be ~0)


## 4. The honest verdict

**Signal — Real (but thin & regime-bound).** The AAA-CLO carry *is* a genuine risk-adjusted pickup: JAAA earns **+1.38%/yr** over cash on just **1.63%** vol (**Sharpe +0.84**, HAC *t* +2.33, bootstrap CI clear of zero), dominating IG corporates (Sharpe -0.11), Treasuries (-0.33) *and* the un-tranched loan collateral (+0.24) — so the seniority earns real keep. ICLO independently confirms (+0.88). **But** it's a regime bet: in the ZIRP era the excess was **+0.00%/yr** (nothing to pocket when cash is 0); all of it is the +1.84%/yr high-rate era. And the ~5.7y sample has **no CLO stress event** — the one AAA-CLO mark-down (Mar-2020) predates JAAA.

**Tradability — Fragile.** It survives costs trivially (buy-and-hold, the ~0.20% ER already in the NAV, tight spreads → net **+1.35%/yr**) and capacity is ample (JAAA is a >$20bn fund). What makes it *Fragile* not *Investable*: the whole ~1.4%/yr IS an insurance premium for a tail (illiquidity/credit) the calm sample never charged — a single crisis mark-down (AAA CLOs fell ~5-10% in Mar-2020) would erase years of carry, so the realized Sharpe flatters the true edge.